vertical-pod-autoscaler/README.md

```diff
@@ -296,6 +296,20 @@ Please note the usage of the following arguments to override default names and p
 
 You can then choose which recommender to use by setting `recommenders` inside the `VerticalPodAutoscaler` spec.
 
+### Override default values when OOM occurs
+The below parameters can be used to overwrite the default values when an OOM event is occured.
+OOMBumpUpRatio specifies how much memory will be added after observing OOM.
```

Comment:
s/OOMBumpUpRatio/oom-bump-up-ratio/
?

Also I'd explain this in some more detail.

When VPA observes an OOM it will use a sample that higher of:
 - `oom-min-bump-up-bytes` B over memory usage observed during OOM and
 - `oom-bump-up-ratio` times the memory usage observed during OOM.